mesh/ncmesh.hpp

```diff
@@ -1025,6 +1025,7 @@ class NCMesh
                          Array<int> *neighbors, /* append */
                          Array<char> *neighbor_set = NULL);
 
+public:
```

Comment:
I'm wondering why `FindNeighbors` and `NeighborExpand` weren't already public.  Do they modify member variables?  If not, perhaps make them `const` member functions would alleviate any concerns with making them `public`